src/data/use-cases/db-create-access-token.ts

```diff
@@ -0,0 +1,36 @@
+import { NotFoundError } from '@/application/errors'
+import {
+  type TokenGenerator,
+  type UpdateAccessTokenRepository,
+  type HashGenerator,
+  type LoadGuardianByEmailRepository
+} from '@/data/protocols'
+import { type CreateAccessToken } from '@/domain/use-cases/create-access-token'
```

Comment:
you could perform the import of "domain/use-cases".

and maintains the order of bringing the domain imports first.

Reply:
first part, about import of "domain/use-cases" make sense, I will fix it. But second part, what order are you refer? I use alphabethic order of path to imports.